src/components/pages/Executors/ExecutorDetails/ExecutorSettings/CommandAndArguments/Command.tsx

```diff
@@ -72,7 +72,7 @@ const Command: React.FC = () => {
           form.resetFields();
         }}
       >
-        <Form.Item label="Command" required name="command" rules={[required]}>
+        <Form.Item label="Command" required name="command" >
```

Comment:
```suggestion
        <Form.Item label="Command" name="command">
```

Reply:
Ok I got that, can I make a new PR for this?

Reply:
You don't have to make a new PR - you may either:
* add a new commit to your branch with this change, or
* click "Commit suggestion" - this way GitHub will automatically add a commit with this suggestion :)